tests/AxisAlignedBBTest.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\math\tests;
+
+
+use pocketmine\math\AxisAlignedBB;
+use pocketmine\math\Vector2;
+use pocketmine\math\Vector3;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class for the AxisAlignedBB class unit test
+ */
+class AxisAlignedBBTest extends TestCase{
+
+	public function testConstructor(){
+		$axis = new AxisAlignedBB(1.1, 1.2, 1.3, 2.1, 2.2, 2.3);
+
+		$this->assertInstanceOf(AxisAlignedBB::class, $axis);
```

Comment:
This seems redundant. `new ...` will _always_ return an instance of that object - we're not working with extensions here.